composer.json

```
@@ -0,0 +1,75 @@
+{
+    "name": "nplants/wp2static-addon-custom-crawling-storage",
+    "description": "Custom Crawling Storage Add-on for WP2Static.",
+    "homepage": "https://wp2static.com",
+      "license": "UNLICENSE",
+      "authors": [
+        {
+          "name": "Natural Plants",
+          "email": "develop@n-plants.biz",
+          "homepage": "https://www.n-plants.co.jp"
+        }
+      ],
+    "type": "wordpress-plugin",
+    "support": {
+        "issues": "https://github.com/naturalplants/wp2static-addon-custom-crawling-storage/issues",
+        "forum": "https://wp2static.com/community",
+        "docs": "https://wp2static.com/documentation",
+        "source": "https://github.com/naturalplants/wp2static-addon-custom-crawling-storage"
+    },
+    "repositories": [
+    ],
+    "require": {
+        "php": ">=7.4"
```

他のaddon合わせで8.0も入れといてもいいかもしれないです。

https://github.com/leonstafford/wp2static-addon-cloudflare-workers/blob/master/composer.json#L26

```suggestion
        "php": "^7.4 || ^8.0",
```